src/views/forms/AccountManagementForms/SourceSelectMenu.tsx

```diff
@@ -87,6 +91,11 @@ export const SourceSelectMenu = ({
       slotBefore: <$Img src={chain.iconUrl ?? undefined} alt="" />,
       [feesDecoratorProp]: getFeeDecoratorComponentForChainId(chain.type),
     }))
+    .filter((chain) => {
+      // only solana chains are supported on phantom
+      if (walletType === WalletType.Phantom && !chain.value.startsWith('solana')) return false;
```

Comment:
[suggestion] would be nice to extract `solana` into a const so we can avoid people making typos

Reply:
✅